client-app/shared/layout/components/header/_internal/bottom-header.vue

```diff
@@ -54,43 +61,71 @@
             </div>
           </template>
         </BottomHeaderLink>
-
-        <i class="fas fa-search text-[color:var(--color-primary)] cursor-pointer" @click="showSearchBar" />
       </div>
     </div>
-
-    <!-- Desktop Search bar -->
-    <div class="absolute top-0 w-full" :class="{ 'overflow-hidden': isAnimatedSearchBar }">
-      <transition
-        enter-from-class="translate-x-full"
-        leave-to-class="translate-x-full"
-        enter-active-class="will-change-transform"
-        leave-active-class="will-change-transform"
-        @enter="isAnimatedSearchBar = true"
-        @leave="isAnimatedSearchBar = true"
-        @after-enter="isAnimatedSearchBar = false"
-        @after-leave="isAnimatedSearchBar = false"
-      >
-        <SearchBar v-show="searchBarVisible" class="w-full transition-transform duration-300" />
-      </transition>
-    </div>
   </div>
 </template>
 
 <script setup lang="ts">
 import { ref } from "vue";
+import { useI18n } from "vue-i18n";
 import { VcImage } from "@/components";
+import Catalog from "./catalog.vue";
 import BottomHeaderLink from "./bottom-header-link.vue";
 import { useCart } from "@/shared/cart";
 import { useSearchBar, SearchBar, useNavigations } from "@/shared/layout";
 import { useCompareProducts } from "@/shared/compare";
 import { useUser } from "@/shared/account";
+const { t } = useI18n();
 
 const { organization } = useUser();
 const { cart } = useCart();
-const { mainMenuLinks } = useNavigations();
+const { desktopCatalog } = useNavigations();
 const { productsIds } = useCompareProducts();
 const { searchBarVisible, showSearchBar } = useSearchBar();
```

Comment:
This row and below one are not needed anymore, please check unused imports/variables.